Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM alpine:3.6
+ARG bashver
```

Comment:
Could we set a default version of `latest` here please?

Reply:
Good call.  This is actually my first time using `ARG` and the documentation I had read didn't make any mention of being able to set default values (though that seems like a no-brainer in retrospect).  Thanks!